Makefile

```diff
@@ -171,6 +171,7 @@ go-run:
 			-tags "$(GO_TAGS)" \
 			./cmd/main.go manager \
 				--development \
+				--enable-leader-election=false \
```

Comment:
This is just to enable quicker iteration when running locally and could be moved to a separate PR